package/features/steps/lower_requirements.txt

```diff
@@ -1,13 +1,14 @@
 ipython==7.0.0
 fastapi==0.100.0
-fsspec==2021.4
+fsspec==2021.06.01
 aiofiles==22.1.0
 uvicorn[standard]==0.22.0
 watchfiles==0.24.0
 plotly==4.8
 packaging==23.0
 pandas==1.3; python_version < '3.10'
-pandas==1.5; python_version >= '3.10'
+pandas==1.5; python_version >= '3.10' and python_version < '3.12'
```

Comment:
maybe we do similar for fsspec too since we don't have cloud accounts to test. only update the version for users who are on 3.12